docs/infrastructure/docker-base-image.md

```diff
@@ -0,0 +1,26 @@
+# The alternatives for Java 11 base docker image
+
+Java 11 is an [LTS version](https://www.oracle.com/java/technologies/java-se-support-roadmap.html). But as Oracle only releases patches for the most recent versions and it has a 6-month release interval for major Java versions, [LTS versions are not eligible for Oracle support, which is effectivelly done by other companies such as Red Hat, Azure (Microsoft), etc](https://github.com/docker-library/official-images/pull/5710).
+
+Oracle releases binaries for what are called OpenJDK distributions. Then, maintainers, spearheaded by RedHat, release patches on the AdoptOpenJDK stream.
+
+This is true for Java versions. As for Docker images, despite images being associated with the same designations (OpenJDK and AdoptOpenJDK), there is a slight difference in who publishes them. OpenJDK docker images are not released by Oracle, but use OpenJDK java distributions to assemble docker images. These are entirely [crowd-sourced](https://github.com/docker-library/openjdk).
+
+As for image size, addition of a [modular architecture](https://www.oracle.com/corporate/features/understanding-java-9-modules.html) in Java 9 – project jigsaw – makes both the jre images more heavy in size. Contrary to what happens with java 8, no linux alpine image exists currently for java 11.
```

Comment:
I don't understand the "more heavy in size"?

Reply:
it was not clear. I updated it.